pkg/minishift/addon/manager/addon_manager.go

```diff
@@ -272,6 +284,32 @@ func (m *AddOnManager) verifyRequiredVariablesInContext(context *command.Executi
 	return nil
 }
 
+func (m *AddOnManager) verifyRequiredOpenshiftVersion(context *command.ExecutionContext, meta addon.AddOnMeta) error {
+	dockerCommander := context.GetDockerCommander()
+	versionInfo, err := dockerCommander.Exec(" ", constants.OpenshiftContainerName, "openshift", "version")
+	if err != nil {
+		return err
+	}
+	openShiftVersionAlongWithCommitSha := strings.Split(strings.Split(versionInfo, "\n")[0], " ")[1]
+	openShiftVersion := strings.Split(openShiftVersionAlongWithCommitSha, "+")[0]
+	requiredOpenshiftVersion := meta.OpenShiftVersion()
+	if requiredOpenshiftVersion != "" {
+		if strings.HasSuffix(requiredOpenshiftVersion, "+") {
```

Comment:
What is the whole '+' business about? You seem to trim it off here. I am not quite following.